### ISSUE TITLE
Unable to start service 

### ISSUE THREAD

**Issue by Shwig** [AndrewBabbitt97/AuraConnect#99]
**System and OS info:**

- OS: Windows 10 Pro
- Version: 2004
- OS Build: 19041.630
- Experience: WFEP 120.2212.31.0
- Asus Rog Strix GTX 1070
- AMD Ryzen 5 1600
- No over clocks

**Method that produced the error:**

1. After initial run of AuraConnectSetup.exe the service was not automatically started.
2. Went to task manager and tried to start the service manually. 
3. Status changes from stopped to starting then back to stopped. 
4. Opened Services app right click on AuraConnect and start.
5. Service Control attempts to start the service and times out.
6. Error message is produced.

**The Error message:**

"Windows could not start the AuraConnect service on Local Computer. 
  Error 1053: The service did not respond to the start or control request in a timely fashion."

**Screen snips:**

**error message**
![auroConnectError](https://user-images.githubusercontent.com/35754177/100747240-c86fde00-33af-11eb-8ae6-23a1bafc711e.png)

**aura in explorer**
![auraSync](https://user-images.githubusercontent.com/35754177/100748196-08839080-33b1-11eb-8f2e-2ba16470646b.png)

**task manager & Services app menu**
![taskManagerAndServices](https://user-images.githubusercontent.com/35754177/100749155-46cd7f80-33b2-11eb-8366-73829fc74e82.png)

**Comment by AndrewBabbitt97** [comment]
Need console output from manually running the program. Sounds like it is probably a COMException, something probably to do with Aura/Armory Crate

**Comment by Shwig** [comment]
> Need console output from manually running the program. Sounds like it is probably a COMException, something probably to do with Aura/Armory Crate

Here is the output from power shell

![image](https://user-images.githubusercontent.com/35754177/100752428-9a41cc80-33b6-11eb-8993-221c54155f29.png)

**Comment by AndrewBabbitt97** [comment]
No as in by manually running AuraConnect.exe from it's install directory, it doesn't have to be run as a service.

**Comment by Shwig** [comment]
> No as in by manually running AuraConnect.exe from it's install directory, it doesn't have to be run as a service.

I added both screen shots, but it seems I've got it to run. My asus arua sync that I was running was installed with the ASUS GPU tweakII.
After installing the asus armory crate it seems to have fixed the problem, it works but it would be cool to not have another lighting utility installed, not a big deal, it's still sweet to be able to have everything synced up. 
I'll close up the issue.

Thank you for your time.
